app/src/main/java/com/cornellappdev/scoop/ui/CalendarApplication.kt

```diff
@@ -0,0 +1,8 @@
+package com.cornellappdev.scoop.ui
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class CalendarApplication: Application() {

```

Comment:
can you rename this to ScoopAplication, this class is for all of the dependency injection in Scoop, not just for the calendar! You need to change this in the manifest as well